src/main/java/io/knowledgeassets/myskills/server/community/query/CommunityQueryController.java

```diff
@@ -45,10 +47,10 @@ public CommunityQueryController(CommunityQueryService communityQueryService,
 	})
 	@PreAuthorize("isAuthenticated()")
 	@GetMapping(path = "/communities", produces = MediaType.APPLICATION_JSON_VALUE)
-	public List<CommunityResponse> getCommunities() {
+	public List<RecommendedCommunityResponse> getCommunities() {
```

Comment:
OK, my second point might not be true today because I assume the plan was to display a single list of communities but with recommended communities sorted to the top. With this in mind, it's really the most obvious solution to have a `recommended` property for each community.

Nevertheless, I think that my first and third point still apply and we should discuss alternative solutions. I see two options so far:

1. We stick with the current implementation, i.e. the API returns a flag for each community to indicate whether it's recommended to the user. I'd prefer to rename the response class back to `CommunityResponse` because we also didn't call it `CommunityWithMembersAndManagersResponse` to reflect that those resources are embedded as well, right? 😉 
2. We implement a separate API to provide community recommendations. In this case, we also should rethink the web app design. I think of a page with a plain list of all communities (as it is currently) and in addition a set of cards above that list with the recommended communities. The recommended communities are also contained in the plain list below. The cards are just like teasers in a app store with a more prominent design to attract attention.

Maybe this has to be discussed with our PO as well. 🤔

Reply:
I moved the discussion about API design to #49 - therefore we can keep this special response class at the moment. Though, my wish is still to have a rather neutral class name like `CommunityResponse` if possible. Maybe we can achieve this with the API redesign.